### PR TITLE
Development

### PR DESCRIPTION
**Pull request by marient**
This version compiles with the PeleC_MP package and Emmanuel's NSCBC.

### CODE REVIEW

**Comment by emotheau** [comment]
Note that the NSCBC and more generally the way we impose boundary conditions have been totally remodeled, so this commit will not be consistent with the incoming changes in the branch GC-NSCBC_cleaning.

**Comment by rgrout** [comment]
Yeah, I thought that might be the case - @emotheau is the latest development merged into your branch? If so @marient can branch off of GC-NSCBC_cleaning and then make that new branch the target for this pull request to keep everything clean.

**Comment by rgrout** [comment]
Other comments:
</ascldap/users/marient/ECP/amrex-master/Src/EB/AMReX_EBFArrayBox.H>  - should not include local paths

CI check doesn’t build - please make sure there are appropriate #ifdef guards so that all the other cases in RegTests still build & pass

**Comment by emotheau** [comment]
Yes the branch GC-NSCBC_cleaning is up to date and ready to be committed in development. Note also that I wrote a documentation of the NSCBC stuff !

**Comment by jrood-nrel** [comment]
It looks like this reverses some changes I made in `Make.PeleC`. Is this necessary for some reason?

File: Exec/Make.PeleC [hunk 1]
```diff
@@ -72,7 +72,6 @@ ifdef Chemistry_Model
   CHEM_HOME = $(PELE_PHYSICS_HOME)/Support/Fuego/Mechanism/Models/$(Chemistry_Model)
   VPATH_LOCATIONS += $(CHEM_HOME)
   Bpack += $(CHEM_HOME)/Make.package
-  Blocs += $(CHEM_HOME)
```
Comment (jrood-nrel) [inline Exec/Make.PeleC]: I don't think this should be removed.

**Comment by drummerdoc** [comment]
It seems like this pull request is obsolete.  If so, can you cancel it so that it doesn't get confused with other PR's?

**Comment by marient** [comment]
Yes, go ahead, thanks.

**Comment by rgrout** [comment]
@marient is there some reason you didn't close this request per the above?

**Comment by drummerdoc** [comment]
Oops, I guess that was on me....  canceling now.

**Pull request closed without merging**